### DISCUSSION TITLE
‘Live’ inbox (realtime updates)

### DISCUSSION THREAD

**Discussion by bgriffith** [community/community#44618] [Mobile] [answered]
I often use the iPad OS app (great work by the way!) and leave the Inbox view open as a way to manage activity across multiple projects and repos. Currently this requires a manual pull down to refresh. 

It would be great if we could have some form of live inbox, where new notifications would appear without the requirement to manually refresh.

Is this currently possible? If not - could it be something for the future? 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.96.0 (88967961)
Device: iPhone 12 (iOS 16.3)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @bgriffith I passed this on to the team for consideration. Thanks!
